### PR TITLE
feat (SWA-243) stop back button from floating

### PR DESCRIPTION
**Pull request by patrinoua**
The type of this PR is: **feat**

[SWA-243] keep back button only on the first screen of submission flow
[SWA-207]

❗️ Please note: 
It seems to me like my prettier has added some unnecessary spaces in lines 22-25.

Maybe you can try locally to see if this happens, if you add an extra prop (eg hasBackButton) in line 22 of **CollapsibleMenuItem** ? 
Thank you in advance 🙏

This PR resolves []

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [ ] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

### Changelog updates

ⓘ 'User-facing' changes will be published as release notes.

#### Cross-platform user-facing changes

- Rename modal title - patrinoua

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

[SWA-243]: https://artsyproduct.atlassian.net/browse/SWA-243?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[SWA-207]: https://artsyproduct.atlassian.net/browse/SWA-207?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 262fbd2463d65fb1b6cec360bb8d6e54085db3be
</p>

File: src/lib/Scenes/Consignments/Screens/SubmitArtworkOverview/SubmitArtworkOverview.tsx [hunk 1]
```diff
@@ -80,38 +82,41 @@ export const SubmitArtworkScreen: React.FC<SubmitArtworkScreenNavigationProps> =
...
-    </ScrollView>
+    <Flex>
+      <ScrollView
+        contentContainerStyle={{
+          paddingVertical: 20,
+          paddingHorizontal: 20,
+          justifyContent: "center",
+        }}
+      >
+        <BackButton onPress={() => goBack()} style={{ top: 10 }} />
```
Comment (MounirDhahri) [inline src/lib/Scenes/Consignments/Screens/SubmitArtworkOverview/SubmitArtworkOverview.tsx:93]: **thought(non-blocker):** I don't think you need to hide the backbutton while scrolling,

Comment (MounirDhahri) [reply src/lib/Scenes/Consignments/Screens/SubmitArtworkOverview/SubmitArtworkOverview.tsx:93]: aha, thanks for linking the requirements Angela

Comment (MounirDhahri) [reply src/lib/Scenes/Consignments/Screens/SubmitArtworkOverview/SubmitArtworkOverview.tsx:93]: Looks clear to me then 🙌

**Review by MounirDhahri — APPROVED** [review]
🙌🙌🙌

**Pull request merged**